Objects/unicodeobject.c

```diff
@@ -15119,16 +15119,13 @@ unicode_subtype_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
 }
 
 PyDoc_STRVAR(unicode_doc,
-"str(object='') -> str\n\
-str(bytes_or_buffer[, encoding[, errors]]) -> str\n\
+"str(object='', encoding='utf-8', errors='strict') -> str\n\
```

Comment:
There is a problem with this. This signature means that `str('', 'utf-8', 'strict')` is equivalent to `str()`. But calling `str` with string argument and encoding is an error. The signature of `str` can't be expressed in such form, this is a cause why the `str` constructor still is not converted to Argument Clinic.

Reply:
I don’t think that merging the two lines is correct.